html/modules/custom/reliefweb_job_tagger/reliefweb_job_tagger.module

```diff
@@ -231,12 +271,8 @@ function reliefweb_job_tagger_entity_after_save(EntityInterface $entity) {
   }
 
   /** @var \Drupal\node\Entity\Node $entity */
-  if ($entity->hasField('reliefweb_job_tagger_status') && $entity->get('reliefweb_job_tagger_status')->value == 'queue') {
+  if ($entity->hasField('reliefweb_job_tagger_status') && $entity->get('reliefweb_job_tagger_status')->value == 'queued') {
```

Comment:
If a user edits a post and save it again, the job is added to the queue again, but the worker will detect it